tests/helpers.bash

```diff
@@ -218,7 +218,7 @@ function k8s_num_ready {
   local NAMESPACE=$1
   local CILIUM_POD=$2
   local FILTER=$3
-  kubectl -n ${NAMESPACE} exec ${CILIUM_POD} cilium endpoint list | grep $FILTER | grep -v 'not-ready' | grep -c 'ready' || true
+  kubectl -n ${NAMESPACE} exec ${CILIUM_POD} cilium endpoint list | grep $FILTER | grep -v -e 'not-ready' -e 'reserved' | grep -c 'ready' || true
   restore_flag $save "e"
 }
```

Comment:
Out of curiosity, if I apply a policy, will the `reserved:health` endpoint go into a regenerating state?

Reply:
The intention is that the API cannot modify policy for reserved endpoints, in which case I would expect not. (The actual restrictions on this aren't in the series yet)